run.sh

```diff
@@ -0,0 +1,20 @@
+set -e
+
+PBF_FILE=$1
```

Comment:
```suggestion
PBF_FILE=${1}
```

I prefer to have variables like this, because that is easier to see 🤓